
#RESTRICTED RIGHTS NOTICE (DEC 2007)
#
# (a)   This computer software is submitted with restricted rights under Government Contract No. DE-AC52-8MA27344 and subcontract B600716. It may not be used, reproduced, or disclosed by the Government except as provided in paragraph (b) of this notice or as otherwise expressly stated in the contract.
#
# (b)   This computer software may be --
#
#       (1) Used or copied for use with the computer(s) for which it was acquired, including use at any Government installation to which the computer(s) may be transferred;
#
#       (2) Used or copied for use with a backup computer if any computer for which it was acquired is inoperative;
#
#       (3) Reproduced for safekeeping (archives) or backup purposes;
#
#       (4) Modified, adapted, or combined with other computer software, provided that the modified, adapted, or combined portions of the derivative software incorporating any of the delivered, restricted computer software shall be subject to the same restricted rights;
#
#       (5) Disclosed to and reproduced for use by support service contractors or their subcontractors in accordance with paragraphs (b)(1) through (4) of this notice; and
#
#       (6) Used or copied for use with a replacement computer.
#
# (c)   Notwithstanding the foregoing, if this computer software is copyrighted computer software, it is licensed to the Government with the minimum rights set forth in paragraph (b) of this notice.
#
# (d)   Any other rights or limitations regarding the use, duplication, or disclosure of this computer software are to be expressly stated in, or incorporated in, the contract.
#
# (e)   This notice shall be marked on any reproduction of this computer software, in whole or in part.


--- ./Makefile	2012-11-27 21:52:46.000000000 -0600
+++ ./Makefile.new	2013-01-28 10:11:03.476853860 -0600
@@ -71,7 +71,7 @@
 endif
 
 # rules verbosity
-ifneq '$V' '0'
+ifneq '$V' '1'
 P = @ true
 E =
 else
@@ -153,27 +153,28 @@
 	$P '  INSTALL DIRS'
 	$E mkdir -p $(DESTDIR)/usr/bin
 	$E mkdir -p $(DESTDIR)/usr/include
-	$E mkdir -p $(DESTDIR)/usr/lib/pkgconfig
+	$E mkdir -p $(DESTDIR)/usr/lib64/pkgconfig
 	$E mkdir -p $(DESTDIR)/usr/share/man/man3
 	$E mkdir -p $(DESTDIR)/usr/share/man/man1
 	$E mkdir -p $(DESTDIR)/usr/share/doc/libroxml/html
 	$P '  INSTALL FILES'
-	$E install -D $(TARGET_SLIB) $(DESTDIR)/usr/lib
-	$E install -D $(TARGET_LIB) $(DESTDIR)/usr/lib
+	$E install -D $(TARGET_SLIB) $(DESTDIR)/usr/lib64
+	$E install -D $(TARGET_LIB) $(DESTDIR)/usr/lib64
 	$E install -D $(TARGET_BIN) $(DESTDIR)/usr/bin
 	$E install -D $(INC) $(DESTDIR)/usr/include
 	$E install -D docs/roxml.1 $(DESTDIR)/usr/share/man/man1/
 	$E [ ! -d docs/man/man3 ] || install -D docs/man/man3/* $(DESTDIR)/usr/share/man/man3/
 	$E [ ! -d docs/html ] || install -D docs/html/* $(DESTDIR)/usr/share/doc/libroxml/html/
-	$E install -m644 libroxml.pc $(DESTDIR)/usr/lib/pkgconfig
-	$E cp -d $(TARGET_LN) $(DESTDIR)/usr/lib
+	$E install -m644 libroxml.pc $(DESTDIR)/usr/lib64/pkgconfig
+	$E cp -d $(TARGET_LN) $(DESTDIR)/usr/lib64
 
 .PHONY: uninstall
 uninstall:
 	$P '  UNINSTALL'
-	$E rm -f $(DESTDIR)/usr/lib/pkgconfig/libroxml.pc
-	$E rm -f $(DESTDIR)/usr/lib/$(TARGET_SLIB)
-	$E rm -f $(DESTDIR)/usr/lib/$(TARGET_LIB)
+	$E rm -f $(DESTDIR)/usr/lib64/pkgconfig/libroxml.pc
+	$E rm -f $(DESTDIR)/usr/lib64/$(TARGET_SLIB)
+	$E rm -f $(DESTDIR)/usr/lib64/$(TARGET_LIB)
+	$E rm -f $(DESTDIR)/usr/lib64/$(TARGET_LN)
 	$E rm -f $(DESTDIR)/usr/bin/$(TARGET_BIN)
 	$E rm -f $(DESTDIR)/usr/include/$(INC)
 	$E rm -fr $(DESTDIR)/usr/share/doc/libroxml
